src/components/Nav.vue

```diff
@@ -23,6 +23,11 @@ const space = computed(() =>
 );
 const { treasury } = useTreasury(space);
 
+const showNavigationItems = computed(() => {
```

Comment:
We should not see the menu area in left and right, the proposal should appear centered on the page and top nav area with Snapshot X should take the whole width (minus sidebar) similar than before. This looks like a trick to hide the menu we should have a more elegant way to do this where we don't run the code of the menu.

Reply:
agreed we need more elegant solution, need to break Nav to NavSettings and NavSpace and control them better. This PR was like a quick patch, but now it will evolve.

Reply:
We should avoid quick patch, we only use quick patch in production to fix something urgent